runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -378,6 +378,8 @@ struct AMDAIEDeviceModel {
   DenseMap<uint32_t, SmallVector<uint32_t>> getChannelToValidBdIds(
       AMDAIETileType tileType) const;
 
+  uint8_t getDmaMaxQueueSize(uint8_t col, uint8_t row);
```

Comment:
I know this class isn't completely organized, but I think it helps to put new functions at the most sensible place in the class. In this case, I think `getDmaMaxQueueSize` fits together with other DMA information retrieval methods, like `getDmaBdProp and `getDmaProp`, so maybe move it after those? Same for the function in .cpp.